docs/index.md

```
@@ -27,29 +31,27 @@ Everything else is out of scope of the library, there are no definitions for "sc
-There are two core components in MVIKotlin: 
+There are two core components in MVIKotlin:
 
 - `Store` - represents `Model` from MVI, this is the place for business logic
 - `MviView` - represents `View` from MVI, the UI part, optional
 
-### How the data flows
+## How the data flows
 
 Please take a look at the following diagram:
 
 <img src="media/mvikotlin.jpg" width="384">
```

markdownlint-cli2 (0.18.1)

Images should have alternate text (alt text)

(MD045, no-alt-text)